ifaddr/_win32.py

```diff
@@ -102,10 +106,10 @@ def get_adapters(include_unconfigured=False):
                                       ctypes.byref(addressbuffer),
                                       ctypes.byref(addressbuffersize))
     if retval != NO_ERROR:
-        raise ctypes.WinError()
+        raise ctypes.WinError()  # type: ignore
```

Comment:
What's the error being silenced here?

Reply:
The Linux version of `ctypes` doesn't contain `WinError`.